apps/projects/templates/a4_candy_projects/project_list.html

```diff
@@ -11,7 +11,7 @@
         {% if filtered_projects|length > 0 %}
             <ul class="l-tiles-4">
                 {% for project in filtered_projects %}
-                    {% include "a4_candy_projects/includes/project_list_tile.html" with project=project %}
+                    {% include "a4_candy_projects/includes/project_list_tile.html" with object=project project=project orientation='vertical' type='project' url=project.get_absolute_url %}
```

Comment:
is `object=project` needed?

Reply:
yes, we rewrote the tile include to use object. So that now they can be used for modules and projects.

Reply:
@khamui I think so as the tile template uses `object.pk` and even if it's not useful in this instance, if we make changes to the tile template relying on list knowing the object we might not catch if it breaks something somewhere